core/src/apps/base.py

```diff
@@ -319,6 +322,11 @@ def set_homescreen() -> None:
 
         set_default(lockscreen)
 
+    elif _SCREENSAVER_IS_ON:
+        from apps.homescreen import screensaver
+
+        set_default(screensaver, restart=True)
```

Comment:
This won't interfere with webauthn when called from fido2.py `set_homescreen` -> `set_default` -> `kill_default`, right?

Reply:
I didn't test that but I don't think it should. `kill_default` is a no-op when a workflow is running, and FIDO is managing its workflows correctly (that we know of)